app/api/test_embedding.py

```
@@ -0,0 +1,238 @@
+        tags=["모델 테스트"],
+        summary="모델 목록 조회",
+        description="테스트할 모델 목록을 반환합니다.")
+async def get_model_list():
+    """테스트할 모델 목록 반환"""
+    return {
+        "status": "success",
+        "models": MODELS_TO_TEST,
+        "total_models": len(MODELS_TO_TEST)
+    }
+
+@router.get(
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)